moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -264,41 +264,6 @@ export const ProjectActivityLogTableMaps = {
       },
     },
   },
-  moped_proj_fund_opp: {
```

Comment:
interesting—so we have activity log handling set up for this table, but the table itself was never implemented?

Reply:
@johnclary I'm glad you asked this question! There were 6 more entries in this object that needed to be removed to align with tables that were dropped. I've pushed up commits to do that into this branch / PR. 

To answer your question, yes, we did have triggers in place in the database for this table to have changes tracked in the activity log. I have audited which tables are tracked by looking for those triggers, and the results are below. Nothing sticks out to me as out of place, but others may have see some that we should enable tracking on.

```
Tracked Tables:
 moped_proj_entities 
 moped_proj_financials
 moped_proj_funding
 moped_proj_notes
 moped_proj_partners
 moped_proj_personnel
 moped_proj_phases
 moped_project
 moped_project_files
 moped_project_types

Untracked Tables:
 moped_activity_log
 moped_city_fiscal_years
 moped_components
 moped_department
 moped_entity
 moped_fund_programs
 moped_fund_sources
 moped_fund_status
 moped_funds
 moped_milestones
 moped_organization
 moped_phases
 moped_proj_components
 moped_proj_components_subcomponents
 moped_proj_features
 moped_proj_features_components
 moped_proj_fiscal_years
 moped_proj_milestones
 moped_project_roles
 moped_status
 moped_subcomponents
 moped_subphases
 moped_types
 moped_users
 moped_workgroup
```

Thanks again!